### ISSUE TITLE
iOS: Sweep paper wallets

### ISSUE THREAD

**Issue by imaginaryusername** [Electron-Cash/Electron-Cash#1296]
Leaving this issue here to keep track now that #562 is closed, no pressure. =D

**Comment by cculianu** [comment]
Good point. Thanks for opening it. I plan on doing this.  Also plan on supporting BIP38 in the iOS UI.

Thanks for opening.

**Comment by CreativeName4** [comment]
Thank you thank you!! this will be my go to IOS wallet as soon as this is an option! 
You guys have done a tremendous job!

**Comment by cculianu** [comment]
I can probably do it very soon (maybe either this weekend or during the week next week) -- it's not that much work.. just some UI screens to add and thoroughly test.

You're welcome!

**Comment by CreativeName4** [comment]
> I can probably do it very soon (maybe either this weekend or during the week next week) -- it's not that much work.. just some UI screens to add and thoroughly test.
> 
> You're welcome!

Thank you, this is great!! I'm really excited for EC robust IOS options!

**Comment by cculianu** [comment]
:D
